### ISSUE TITLE
Messenger - Dark Mode white on white

### ISSUE THREAD

**Issue by sethmayer** [ferdium/ferdium-app#1424]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that Ferdium Recipes adheres to.
- [X] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22recipe+%3Abookmark_tabs%3A%22+label%3A%22bug+%3Abeetle%3A%22+) for a bug report that matches the one I want to file, without success.

### Ferdium Version

6.5.2

### Recipe Name

Messenger

### Recipe Version

1.8.0

### Last Known working version of the recipe

1.8.0

### Steps to reproduce

Enable Dark Mode for Messenger
Browse to Chat
See that messages have light background and light text

### Expected Behavior

Messages should have contrasting background and text to remain visible

### Actual Behavior

Inbound messages are colored `#cecece`

### Screenshots

![image](https://github.com/ferdium/ferdium-app/assets/5420660/5c9b8047-f132-4e06-a232-33d7a91f0bef)

### Additional Information

This just appeared randomly on the morning of 10/25, and has been confirmed with others using Ferdium. I suspect the issue is with a change on the service side. I've worked around the issue by chaging the `--primary-text:` color in `darkmode.css` but it still doesnt look right

**Comment by tamas646** [comment]
I've added `--chat-incoming-message-bubble-background-color: #292929;` to `:root, .__fb-light-mode` in the css file.
With this it looks nice.

**Comment by tamas646** [comment]
Update:
With `--chat-incoming-message-bubble-background-color: #333;` it looks even nicer. :)
Maybe this was the original color.

**Comment by SpecialAro** [comment]
Hello

I cannot stress this enough but please do not open duplicates of issues before looking at the issue board for similar issues.

This is a duplicate of #1416

**Comment by sethmayer** [comment]
Thanks @SpecialAro, didn't mean to make your life more difficult. I guess I just suck at searching.
